### ISSUE TITLE
About Internal feature distance for shift Equivariance visualization

### ISSUE THREAD

**Issue by re20100801** [adobe/antialiased-cnns#41]
Dear Author:
     Thank you for sharing your insight and code. I have a question as below
     For the Shift Equivariance visualization in Figure 5, it looks like the feature map always has same resolution with input image(32x32), how could you achieve that? As I know, the resolution of feature map should reduce as downsampling is executed.
     If the bilinear interpolation is used, is that a fair feature distance metrics?
     A more general question is how could one measure shift-equivariance between pixel level shift on input image and corresponding sub-pixel shift on feature map 
     Thank you in advance!

**Comment by richzhang** [comment]
Thanks for the question. I upsampled the feature maps back to full resolution (Fsquiggle). Indeed this wasn't documented fully.

I upsampled the intermediate feature maps using bilinear upsampling with circular padding. For example, a length 4 signal (A,B,C,D) --> (A, AB, B, BC, C, CD, D, DA), where AB means (A+B)/2. For factors of 4,8,16, etc, I upsampled 2x recursively until reaching the full feature map size.
